drip/drips.py

```diff
@@ -195,6 +195,24 @@ def walk(self, into_past: int = 0, into_future: int = 0):
         return walked_range
 
     def apply_queryset_rules(self, qs: str) -> str:
+        return (
+            self.apply_and_queryset_rules(qs) |
+            self.apply_or_queryset_rules(qs)
+        )
+
+    def apply_or_queryset_rules(self, qs: str) -> str:
+        query = None
+        rule_set = self.drip_model.queryset_rules.filter(rule_type='or')
+        for rule in rule_set:
+            kwargs = rule.filter_kwargs(qs, now=self.now)
+            query_or = Q(**kwargs)
+            if query is None:
+                query = qs.none()
+            query = query | query_or
+        qs = qs.filter(query) if query is not None else qs.none()
```

Comment:
This line could be:

```python
qs = qs.filter(query) if query else qs.none()
```

Reply:
I think in this case having the `is None` actually adds clarity to what query can be.

Reply:
Sounds good to me.